bench-vortex/Cargo.toml

```diff
@@ -29,6 +29,7 @@ datafusion = { workspace = true, features = [
     "parquet",
     "datetime_expressions",
     "nested_expressions",
+    "unicode_expressions",
```

Comment:
I'm not sure why this feature wasn't needed before, as tpch q22 uses `substring`.

I posted about in the DF release thread so maybe we'll find an explanation.